src/Custom/Embeddings/OpenAIEmbeddingModelFactory.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenAI.Embeddings;
+
+/// <summary> Model factory for models. </summary>
+public static partial class OpenAIEmbeddingModelFactory
+{
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Embeddings.Embedding"/>. </summary>
+    /// <param name="index"> The index of the embedding in the list of embeddings. </param>
+    /// <param name="vector"> The embedding vector, which is a list of floats. </param>
+    /// <returns> A new <see cref="OpenAI.Embeddings.Embeddings"/> instance for mocking. </returns>
+    public static Embedding Embedding(int index = default, IEnumerable<float> vector = null)
+    {
+        vector ??= new List<float>();
+
+        return new Embedding(
+            index,
+            vector.ToArray());
+    }
+
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Embeddings.EmbeddingCollection"/>. </summary>
+    /// <param name="items"> The list of embeddings generated by the model. </param>
+    /// <param name="model"> The name of the model used to generate the embedding. </param>
+    /// <param name="usage"> The usage information for the request. </param>
+    /// <returns> A new <see cref="OpenAI.Embeddings.EmbeddingCollection"/> instance for mocking. </returns>
+    public static EmbeddingCollection EmbeddingCollection(IEnumerable<Embedding> items = null, string model = null, EmbeddingTokenUsage usage = null)
```

Comment:
I'm using the name `items` for elements in the collection. I'm not sure whether this is a good idea or not. Any chance of these models getting a property name `Items` in the future?

Reply:
`items` is good. `EmbeddingCollection` inherits from `ReadOnlyCollection`, which has the `Items` property.